tests/sklearn/test_fhe_training.py

```diff
@@ -133,27 +133,23 @@ def test_fit_error_if_non_binary_targets(n_bits, max_iter, parameter_min_max):
         model.partial_fit(x, y, fhe="disable")
 
 
-@pytest.mark.parametrize("loss", ["log_loss", "modified_huber"])
```

Comment:
here we are testing error raises for specific arguments, we don't need all these inputs